rxbinding-design/build.gradle

```diff
@@ -42,6 +42,7 @@ android {
   lintOptions {
     textReport true
     textOutput 'stdout'
+    disable 'ResourceType', 'PrivateResource'
```

Comment:
Also even when running `./gradlew :rxbinding-design:lint` on master branch I get the same lint errors.